readeventsfile.go

```diff
@@ -16,18 +15,7 @@ var (
 	ErrInvalidV3Event = errors.New(`json is not in expected v3event format`)
 )
 
-// readArgs0File assumes the first program argument is a file name and reads it
-func readArgs0File(cCtx *cli.Context) ([]logverification.VerifiableEvent, error) {
-	if cCtx.Args().Len() < 1 {
-		return nil, fmt.Errorf("filename expected as first positional command argument")
-	}
-	return ReadVerifiableEventsFromFile(cCtx.Args().Get(0))
-}
-
-func readArgs0FileOrStdIoToVerifiableEvent(cCtx *cli.Context) ([]logverification.VerifiableEvent, error) {
-	if cCtx.Args().Len() > 0 {
-		return ReadVerifiableEventsFromFile(cCtx.Args().Get(0))
-	}
+func stdinToVerifiableEvent(cCtx *cli.Context) ([]logverification.VerifiableEvent, error) {
```

Comment:
much better name, but can we have the plural? `stdinToVerifiableEvents`?

Same for decoded events variant